functests/common.go

```diff
@@ -57,3 +73,44 @@ func GetRandomPVC(storageClass string, quantity string) *k8sv1.PersistentVolumeC
 
 	return pvc
 }
+
+func GetDataValidatorJob(pvc string) *k8sbatchv1.Job {
```

Comment:
@xenolinux this function needs a comment starting with its name since this is an exported function.

Reply:
Added the comment.